### ISSUE TITLE
[BUG] VERY CRITICAL - Saving results in empty OAS file

### ISSUE THREAD

**Issue by nbrugger-tgm** [stoplightio/studio#409]
**Describe the bug**
I have a OAS file and I added a few paths with Stoplight studio. Then i pressed `STRG` (`CTRL` on english keyboard) + `S` and closed Stoplight with the Cross in the corner after the whie dot beside the spec file was gone.

As i tried to generate a HTML out of it the file was empty. In panic i reopened Stoplight but it also told me the file is empty. (a look into the file with Notepad ++ confirmed that)

So basically Stoplight destroyed an entire ~2000 line file

**To Reproduce**
Not more info than above because i don't know whats the reason for this

**Expected behavior**
A error message informing me why the save cannot be saved. And not overwrite the file empty

**Screenshots**

**Environment (remove any that are not applicable):**
 - Studio version: 1.11.0
 - OS: Windows 10
 - Browser: I used desktop version

**Comment by P0lip** [comment]
Hey @nbrugger-tgm
My sincere apologies for any inconvenience caused. Es tut mir wirklich sehr leid :disappointed: 
Data loss is always painful and under no circumstances should happen.
There is little chance the content has been retained in the stash store, therefore let's verify it.
Assuming you still have the project in Studio, could you please open dev tools as shown below
![image](https://user-images.githubusercontent.com/9273484/81347840-39bb0d00-90bd-11ea-9c4b-bcdcafa67f63.png) and execute the following piece of code?
`{ ...__SL.activeProjectStore._cache.graphStash.get() }`
Does the returned object have any properties or is it entirely empty?

I'll ask a few questions now to get a grasp of the whole situation.

- Did you have auto-save enabled? It's turned on by default, but did you disable it by a chance? If auto-save was not disabled, has the file been saved before or was it the first attempt?
- Did you leverage primarily / only the form view to design the API, or was it code editor?
- Did you close the Studio right away after pressing `STRG+S`?

**Comment by nbrugger-tgm** [comment]
## Recovery
With the VCS used and a File Recovery Software wich operates on hardware level (i guess) we got back most of the progress. Half of the sprint is gone but this is ok so far.
## Help for the bug
> Did you have auto-save enabled
Yes
> Did you leverage primarily / only the form view to design the API, or was it code editor?
I use the code editor too but this time (the whole session) i only used the form/gui editor
> Did you close the Studio right away after pressing STRG+S?
Yes, the time between `STRG+S` was aproxemately 1-3 secs. Anyway i did waited for the white point right to my OAS file to disappear
